docker/docker-compose-without-neo4j.postgres.override.yml

```diff
@@ -0,0 +1,76 @@
+# Override to use PostgreSQL as a backing store for datahub-gms.
+---
+version: '3.9'
+services:
+  datahub-gms:
+    env_file:
+      - datahub-gms/env/docker-without-neo4j.env
+      - datahub-gms/env/docker.postgres.env
+    environment:
+      - DATAHUB_SERVER_TYPE=${DATAHUB_SERVER_TYPE:-quickstart}
+      - DATAHUB_TELEMETRY_ENABLED=${DATAHUB_TELEMETRY_ENABLED:-true}
+    depends_on:
```

Comment:
some of these depends on and env rules can go in the base docker-compose file right?

Reply:
Probably, but I didn't want to make any unnecessary changes, especially here where idk what these environment variables are doing / how to test if they work. This is just from copying what's in docker-compose-without-neo4j.override.yaml since I want the postgres version to function like the mysql version, as much as possible